packages/app/src/systems/Account/components/AccountList/AccountList.tsx

```diff
@@ -8,14 +8,16 @@ import { AccountItem } from '../AccountItem';
 export type AccountListProps = {
   accounts?: Account[];
   isLoading?: boolean;
-  onPress?: (account: Account) => void;
+  onPress: (account: Account) => void;
```

Comment:
@LuizAsFight thought we wanted `onPress` to be optional?
```suggestion
  onPress?: (account: Account) => void;
```

Reply:
you're right